lib/fluent/plugin/filter_process_redfishalert.rb

```diff
@@ -11,9 +11,12 @@ class ProcessRedfishAlert < Filter
     config_param :coloregion, :string
     config_param :username, :string
     config_param :passwordFile, :string
+    config_param :hardware, :string, :default => "SDFLEX"
 
     def configure(conf)
       super
+        @hwtSNPartition = Hash["Dell_PowerEdge_iDRAC"=>"Systems/System.Embedded.1", "SDFLEX" => "Chassis/RMC"]
```

Comment:
I don't think partition is a good name for these, in the SD flex case we're not querying the partition. 
Maybe device or something similar?

Reply:
Sure!